antarest/study/storage/variantstudy/variant_study_service.py

```diff
@@ -732,6 +732,16 @@ def _generate_study_config(
         # Generate
         return self._generate_config(metadata, parent_config)
 
+    def get_icommands(
+        self, variant_study_id: str, params: RequestParameters
+    ) -> List[ICommand]:
+        variant_study = self._get_variant_study(variant_study_id, params)
+        commands: List[List[ICommand]] = self._to_icommand(variant_study)
+        commands_flatten = [
+            command for command_list in commands for command in command_list
+        ]
+        return commands_flatten
```

Comment:
would not be needed if we just take all the commands in database (independently of the variant studies)